pattern_generator:
    update_rate: 10 # [Hz]
    number_of_steps_needed: 5
    republish_complete_step_plan: False

    limits:
      x:    { min: -0.1, max: 0.2 }
      y:    0.1
      yaw:  0.1

joystick:
    x:      { axis: 1, thresh: 0.2, min: -1.0, max: 1.0, zero: 0.0, calib: 0.0, invert: False }
    y:      { axis: 0, thresh: 0.2, min: -1.0, max: 1.0, zero: 0.0, calib: 0.0, invert: False }
    yaw:    { axis: 3, thresh: 0.2, min: -1.0, max: 1.0, zero: 0.0, calib: 0.0, invert: False }
    enable: { axis: 2, thresh: 1.0, min: -1.0, max: 1.0, zero: 1.0, calib: 0.0, invert: False }
